test/data.jl

```diff
@@ -42,18 +34,24 @@ module TestData
     # lots more to do
 
     #test_group("assign")
-    df6[3] = @data(["un", "deux", "troix", "quatre"])
+    df6[3] = ["un", "deux", "trois", "quatre"]
```

Comment:
lol

Reply:
That was really a shame for all JuliaStats! ;-)

Reply:
C'est dommage!